tests/unit/amo/actions/test_landing.js

```diff
@@ -1,84 +1,125 @@
-import { getLanding, loadLanding } from 'amo/actions/landing';
+import {
+  getLanding,
+  getLandingForCategory,
+  loadLanding,
+} from 'amo/actions/landing';
 import { ADDON_TYPE_THEME } from 'core/constants';
 
 
-describe('LANDING_GET', () => {
-  const getActionParams = () => ({
-    addonType: ADDON_TYPE_THEME,
-    errorHandlerId: 'some-error-handler',
-  });
-  const action = getLanding(getActionParams());
-
-  it('sets the type', () => {
-    expect(action.type).toEqual('LANDING_GET');
-  });
-
-  it('sets the filters', () => {
-    expect(action.payload).toEqual({
+describe(__filename, () => {
```

Comment:
I have added a new `describe` block with `__filename`, that is why there is a larger diff than expected in this file.